modules/travel_pay/config/routes.rb

```diff
@@ -5,7 +5,7 @@
     resources :claims
 
     scope '/claims/:claim_id', constraints: { claim_id: %r{[^/]+} } do
-      resources :documents, only: %i[index show create]
+      resources :documents, only: %i[index show create destroy]
```

Comment:
Now that we've got most of the CRUD resources here, do we need the `only` or is it fine to just have `resources :documents` like claims above? I'm asking this earnestly. I'm not sure if there's problems with that approach.